test/utils/HyperdriveUtils.sol

```diff
@@ -1233,19 +1233,27 @@ library HyperdriveUtils {
         }
 
         // Calculate the pool's solvency after opening the short.
-        uint256 shareReserves = _params.shareReserves -
-            (shareAmount -
-                (calculateShortCurveFee(
-                    _shortAmount,
-                    _spotPrice,
-                    _params.curveFee
-                ) -
-                    calculateShortGovernanceFee(
-                        _shortAmount,
-                        _spotPrice,
-                        _params.curveFee,
-                        _params.governanceLPFee
-                    )).divDown(_params.vaultSharePrice));
+        uint256 tradeFees = (calculateShortCurveFee(
```

Comment:
Let’s use “curveFee” to be consistent.

Reply:
it is both the curveFee and the governanceCurveFee, which are treaded differently [elsewhere](https://github.com/delvtech/hyperdrive/blob/02efffd127f197629641c91d5baaae61b7247968/contracts/src/internal/HyperdriveBase.sol#L777-L789). With that in mind do you st ill want to use "curveFee"?

Reply:
updating to `totalCurveFee` per outside discusison.